components/collapsible-panel/collapsible-panel.js

```diff
@@ -298,6 +287,21 @@ class CollapsiblePanel extends RtlMixin(LitElement) {
 		`;
 	}
 
+	updated(changedProperties) {
+		if (changedProperties.has('expanded')) {
+			if (!this.expanded) {
+				const element = this.shadowRoot.querySelector('.d2l-collapsible-panel');
+				element.classList.remove('scrolled');
```

Comment:
Hmm, this is breaking the "render() is a function of state" rule.

Suggestion: make an `isScrolled` as a state-based property. Set it to `true` when scrolling happens and `false` here, and then set the `scrolled` CSS class on the `div` inside of `render()` based on its value.

Reply:
Ahh, good call. I was avoiding having two equivalent values (css class + state property), but I guess it's fine to have both 😄